pkg/agent/agent_windows.go

```diff
@@ -218,6 +289,74 @@ func (i *Initializer) prepareOVSBridgeOnHNSNetwork() error {
 	return nil
 }
 
+func (i *Initializer) prepareOVSConfigForVM() error {
+	klog.InfoS("Performing OVS configuration", "hostIfName", i.nodeConfig.UplinkNetConfig.Name)
+	hostIfName := i.nodeConfig.UplinkNetConfig.Name
+	uplinkIfName := util.GenerateUplinkInterfaceName(hostIfName)
+	ovsPorts, ovsErr := i.ovsBridgeClient.GetPortList()
+	if ovsErr != nil {
+		klog.Errorf("Failed to list OVS ports")
+		return ovsErr
+	}
+	for index := range ovsPorts {
+		port := &ovsPorts[index]
+		if port.Name == hostIfName {
+			klog.Info("Uplink and host configuration exists in OVS")
+			return nil
+		}
+	}
+
+	success := false
+	uplinkExternalIDs := map[string]interface{}{
+		interfacestore.AntreaInterfaceTypeKey: interfacestore.AntreaUplink,
+	}
+	// TODO Anand: Have a separate function for creation of pair ports
+	// Create uplink port on OVS.
+	uplinkUUID, ovsErr := i.ovsBridgeClient.CreatePort(uplinkIfName, uplinkIfName, uplinkExternalIDs)
+	if ovsErr != nil {
+		return fmt.Errorf("failed to create uplink port on OVS for %s: %v", uplinkIfName, ovsErr)
+	}
+
+	defer func() {
+		if !success {
+			klog.InfoS("Exiting!! delete port on OVS", "uplinkUUID", uplinkUUID)
```

Comment:
We do not exit in this func right. Probably remove "Exiting" here, also otherwise, we can see "Exiting" multiple times by each these rollback funcs.

Reply:
delete -> Delete

Reply:
As part of Agent initialisation, If there is an error then agent will terminate. That is the reason, I added Exiting.
These functions by themselves do not trigger exit.  I changed the log message to remove "Exiting!!" at both places.

I updated the code to log the errors in case of failures.